drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -172,12 +172,12 @@ void at86rf2xx_set_page(at86rf2xx_t *dev, uint8_t page)
 
 uint16_t at86rf2xx_get_pan(at86rf2xx_t *dev)
 {
-    return dev->pan;
+    return byteorder_ntohs(dev->pan);
 }
 
 void at86rf2xx_set_pan(at86rf2xx_t *dev, uint16_t pan)
 {
-    dev->pan = pan;
+    dev->pan = byteorder_htons(pan);
```

Comment:
isn't 802.15.4 PAN id supposed to be in little endian byte order (i.e. the opposite of "network byte order")?